sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/ContainerSASPermission.java

```diff
@@ -77,108 +77,108 @@ public static ContainerSASPermission parse(String permString) {
     /**
      * @return the read permission status
      */
-    public boolean getRead() {
-        return read;
+    public boolean getReadPermission() {
+        return readPermission;
     }
 
     /**
      * Sets the read permission status.
      *
-     * @param read Permission status to set
+     * @param readPermission Permission status to set
      * @return the updated ContainerSASPermission object
      */
-    public ContainerSASPermission setRead(boolean read) {
-        this.read = read;
+    public ContainerSASPermission setReadPermission(boolean readPermission) {
```

Comment:
Hasn't the name for the parameter been `hasXPermission`?

Reply:
Done.